test/scales/quantitativeScaleTests.ts

```diff
@@ -0,0 +1,56 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Scales", () => {
+  describe("Quantitative Scales", () => {
+    it("domain can't include NaN or Infinity", () => {
+      var scale = new Plottable.Scales.Linear();
+      scale.domain([0, 1]);
+      scale.domain([5, Infinity]);
+      assert.deepEqual(scale.domain(), [0, 1], "Infinity containing domain was ignored");
+      scale.domain([5, -Infinity]);
+      assert.deepEqual(scale.domain(), [0, 1], "-Infinity containing domain was ignored");
+      scale.domain([NaN, 7]);
+      assert.deepEqual(scale.domain(), [0, 1], "NaN containing domain was ignored");
+      scale.domain([-1, 5]);
+      assert.deepEqual(scale.domain(), [-1, 5], "Regular domains still accepted");
+    });
+
+    it("custom tick generator", () => {
+      var scale = new Plottable.Scales.Linear();
+      scale.domain([0, 10]);
+      var ticks = scale.ticks();
+      assert.closeTo(ticks.length, 10, 1, "ticks were generated correctly with default generator");
+      scale.tickGenerator((scale) => scale.getDefaultTicks().filter(tick => tick % 3 === 0));
+      ticks = scale.ticks();
+      assert.deepEqual(ticks, [0, 3, 6, 9], "ticks were generated correctly with custom generator");
+    });
+  });
+
+  describe("Linear", () => {
```

Comment:
Seems like the tests above it also use a `Linear` scale, since currently it's not possible to test with a `QuantitativeScale` directly.